### PR TITLE
gpu: enable gpu if the device is initialized properly

### PR DESCRIPTION
**Pull request by rootfs**

### CODE REVIEW

File: pkg/power/accelerator/source/gpu_nvml.go [hunk 1]
```diff
@@ -57,7 +57,7 @@ func (n *GPUNvml) Init() (err error) {
 		err = fmt.Errorf("failed to get nvml device count: %v", nvml.ErrorString(ret))
 		return err
 	}
-	klog.V(1).Infof("found %d gpu devices\n", count)
+	fmt.Printf("found %d gpu devices\n", count)
 	devices = make([]interface{}, count)
```
Comment (jichenjc) [inline pkg/power/accelerator/source/gpu_nvml.go:61]: we agreed to switch from fmt to klog
why need this change back?

Comment (rootfs) [reply pkg/power/accelerator/source/gpu_nvml.go:62]: this is called in `init` function, klog is not initialized yet so klog Infof doesn't print anything at this time.

Comment (jichenjc) [reply pkg/power/accelerator/source/gpu_nvml.go:62]: ok, thanks for point that out

**Pull request merged**